3rdparty/ErrorHandling/errorcodes.h

```diff
@@ -0,0 +1,29 @@
+#include <map>
+
+struct ErrorCodes
+{
+    enum Errors {
+        UnsupportedFormat,
+        FileNotFound,
+        Blank,
+        ParsemzXml,
+        ParsemzMl,
+        ParsemzData,
+        ParseCsv
+    };
+
+};
+
+//singleton
+class ErrorMsg
```

Comment:
@rish9511 Why do we need to have ErrorCodes class along with MavenException class? Is there a specific reason for both of them being separate? can we combine them?

Reply:
The input to MavenException class is the type of Error that can occur in ElMaven. Depending on the type of Error received, MavenException class may decide to act differently(To be done in future). For e.g,  if the type of the error received is a CRITICAL error, it will log the error differently compared to when the type will be DEBUG. 
Point being, both Classes have a different role to play 
Also, in future, we will  add more error types in the ErrorCode class.

Therefore I believe it's better if we keep them separate

Does this make sense ?

Reply:
@rish9511 Makes sense. Thanks for the explanation.